core/modules/dashboard/includes/block.overview_user.inc

```diff
@@ -0,0 +1,77 @@
+<?php
+/**
+ * @file
+ *
+ * User accounts block which displays a summary of user statistics including:
+ * - number of total user accounts.
+ * - number of active user accounts.
+ * - number ofblocked user accounts.
+ * - number of users in each role.
+ */
+class DashboardOverviewUserBlock extends Block {
+
+  /**
+   *  Sets block subject on block view.
+   */
+  function getTitle() {
+    return !empty($this->settings['title']) ? check_plain($this->settings['title']) : t('User accounts');
+  }
+
+  /**
+   * 'Admin title' callback for the content type.
+   */
+  function getAdminTitle() {
+    return t('Overview of user accounts');
+  }
+
+  /**
+   * 'Admin info' callback for the content type.
+   */
+  function getAdminPreview() {
+    return t('Displays a summary of user statistics including: number of total users, active users, blocked users, and users in each role.');
+  }
+
+  /**
+   * Run-time rendering of the body of the block.
+   */
+  function getContent() {
+    $user_data = dashboard_get_user_overview($this->settings);
+
+    $overview = array();
+    $overview['all'] = format_plural($user_data['all'], '1 user account', '@count total user accounts');
+    $overview['active'] = format_plural($user_data['active'], '1 active user account', '@count active user accounts');
+    $overview['blocked'] = format_plural($user_data['blocked'], '1 blocked user account', '@count blocked user accounts');
+
+    foreach ($user_data['roles'] as $rid => $data) {
+      $overview['roles_' . $rid] = format_plural(
+        $user_data['roles'][$rid]['count'],
+        '1 user account with the role @role',
+        '@count user accounts with the role @role',
+        array('@role' => check_plain($user_data['roles'][$rid]['name'])));
+    }
+
+    return theme('dashboard_overview_user', array('overview' => $overview));
+  }
+
+  /**
+   * 'Edit form' callback for the content type.
+   */
+  function form(&$form, &$form_state) {
+    parent::form($form, $form_state);
+
+    // Add user settings to form.
+    dashboard_add_user_block_settings($form, $this->settings);
```

Comment:
Another function that is only used in one place.

Reply:
fixed by @quicksketch